promptulate/tools/sleep/tool.py

```diff
@@ -1,21 +1,18 @@
 import re
 from time import sleep
 
-from promptulate.tools import BaseTool
+from promptulate import Tool
 
 
-class SleepTool(BaseTool):
-    name: str = "sleep"
-    description: str = (
-        "Make agent sleep for a specified number of seconds."
-        "Input is a number. eg: Sleep for 5s and enter 5"
-    )
+def SleepTool(sleep_time: str) -> str:
```

Comment:
The name does not conform to the function definition specification.The use of functional declarations should follow the corresponding development paradigm. `ssleep is enough.

Reply:
sleep is python time package function

Reply:
sleep_tool is better